src/main/java/ua/edu/internship/interview/data/repository/InterviewRepository.java

```diff
@@ -0,0 +1,16 @@
+package ua.edu.internship.interview.data.repository;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.mongodb.repository.Query;
+import ua.edu.internship.interview.data.documents.InterviewDocument;
+
+import java.time.LocalDateTime;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.time.LocalDateTime'
<!-- __reviewdog__:ChA2NmE4ZTYwMjg0NDQ5OTk3EgpjaGVja3N0eWxl -->